docs/en/troubleshooting.md

```diff
@@ -91,3 +91,42 @@ This restriction is a security feature coming with Plesk 10.
 On round about page 150 of the plesk Administrator Guide you will find a solution to enable scheduled tasks which use the command line. (But the latest Guide for 10.3.1 mentions /usr/local/psa/admin/bin/server_pref -u -crontab-secure-shell "/bin/sh" although "server_pref" doesnt exit.
 Since we are using a dedicated server for only one customer, we defined the crons under "Server Management"->"Tools & Utilities"->"Scheduled Tasks"->"root". The security restrictions of plesk are not involved then. 
 
+## Broken jobs
+
+Sometimes, jobs break without having any issues with their implementation but rather an external factor is the root cause, for example database table lock may prevent a DB write.
+Most common scenario is "publish" action related jobs such as scheduled publish feature which may experience DB deadlocks on the versioned table as this can be frequently accessed.
+
+For this scenario it's recommended to configure automatic job retries.
+Configuration has some flexibility on number of retries and the timing of the retry attempts.
+Use the sample configuration below as a starting point and adjust as needed.
+This code snippet need to be placed into your job class or alternatively placed in a YAML configuration file.
+
+```php
+private static array $broken_status_retry = [
+    // First retry attempt - Retry immediately
+    [
+        0, // <-- offset
+        0, // <-- spread
```

Comment:
What does "offset" do?
What does "spread" do?

Reply:
Updated the docs to have this information.